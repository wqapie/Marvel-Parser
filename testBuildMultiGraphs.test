# Tests building multiple graphs that contains the same hero or heroes in the same book, and verify that they
# cannot "reach" each other.

LoadGraph g1 buildMultiGraphs1.tsv

LoadGraph g2 buildMultiGraphs2.tsv

LoadGraph g3 buildMultiGraphs3.tsv

ListNodes g1

ListNodes g2

ListNodes g3

FindPath g1 Robin Robin3

FindPath g3 Robin Robin3

FindPath g1 Robin Robin

FindPath g2 Robin Robin