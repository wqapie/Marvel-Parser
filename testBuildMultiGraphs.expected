# Tests building multiple graphs that contains the same hero or heroes in the same book, and verify that they
# cannot "reach" each other.

loaded graph g1

loaded graph g2

loaded graph g3

g1 contains: Robin

g2 contains: Robin

g3 contains: Robin3

unknown character Robin3

unknown character Robin

path from Robin to Robin:

path from Robin to Robin: